homeassistant/components/verisure/__init__.py

```diff
@@ -90,10 +90,15 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     HUB.update_overview = Throttle(config[DOMAIN][CONF_SCAN_INTERVAL])(
         HUB.update_overview
     )
+
+    HUB.update_firmware_status = Throttle(config[DOMAIN][CONF_SCAN_INTERVAL])(
+        HUB.update_firmware_status
+    )
```

Comment:
Are you sure you want to update the firmware status at the same interval as updating the hub itself? Seems like more of a once a day type operation?

Reply:
Yes, this is what I said in my first post, It could be reduced, once a day sounds good